### DISCUSSION TITLE
登录授权问题反馈

### DISCUSSION THREAD

**Discussion by hn-kz** [getlantern/lantern-forum-cn#1101] [General]
你好，我安卓端购买了半年的会员，但是没有让我通过邮箱注册啊？目前WINDOWS端我下载好了，找不到授权的地方，目前仅安卓端能用，WINDOWS端目前无法使用

**Comment by MaxMKMax** [comment]
https://lantern.io/zh-Hans/faq
您好. 这个链接应该可以帮到您的. 您试试看. 如果不能的话再回来给我留言, 我看能怎么帮您. 谢谢您支持蓝灯!
